hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/HazelcastConnectorTest.java

```diff
@@ -262,4 +270,82 @@ public void when_readList_and_writeList() throws ExecutionException, Interrupted
         assertEquals(ENTRY_COUNT, jetInstance.getList(sinkName).size());
     }
 
+    @Test
+    public void when_readMap_and_updateMap_with_update_function() throws ExecutionException, InterruptedException {
```

Comment:
these tests seem redundant with their equivalents in Pipeline API. They should all use Pipeline API and preferable be in the same place.